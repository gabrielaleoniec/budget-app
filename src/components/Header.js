import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = () => (
    <div>
        <h1>Expensify App</h1>
        <nav>
            <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
            <NavLink to="/create" activeClassName="is-active">Create</NavLink>
            <NavLink to="/edit" activeClassName="is-active">Edit</NavLink>
        </nav>
    </div>
);

export default Header;