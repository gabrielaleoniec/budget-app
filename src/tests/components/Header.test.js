import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('Should render Header correctly', () => {
    const wrapper = shallow(<Header/>);
    expect(wrapper.find('h1').text()).toBe('Expensify App');
    expect(wrapper).toMatchSnapshot();
});
