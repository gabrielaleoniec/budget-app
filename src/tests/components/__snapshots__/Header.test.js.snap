// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<div>
  <h1>
    Expensify App
  </h1>
  <nav>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/create"
    >
      Create
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/edit"
    >
      Edit
    </NavLink>
  </nav>
</div>
`;
